### PR TITLE
Update Bundle Page text to clarify what to upload and purpose of the page

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Update navigation path and page title
Refactor OperatorEditorPage to operatorBundlePage

### CODE REVIEW

File: frontend/src/pages/documentation/Contribute.js [hunk 1]
```diff
@@ -73,27 +72,27 @@ const Contribute = ({ history, ...props }) => {
...
           <p>
             The easiest way to package your Operator for OperatorHub.io is to build it with the{' '}
             <ExternalLink href={operatorSdk} text="Operator SDK" indicator={false} />, however it is not a requirement.
             To publish your Operator, it must be built as a binary within a container, and that container must be hosted
-            on a publicly accessible{' '}
-            <ExternalLink href={operatorRegistry} text="registry" indicator={false} />. It should be accompanied by
-            some{' '} <ExternalLink href={operatorsRepoRequirements} text="metadata" indicator={false} /> that is
-            used for deploying the Operator using the{' '}
+            on a publicly accessible <ExternalLink href={operatorRegistry} text="registry" indicator={false} />. It
+            should be accompanied by some{' '}
```
Comment (jozzi) [inline frontend/src/pages/documentation/Contribute.js]: wouldn't it be nicer to use &nbsp; instead of {' '} to force space?

Comment (jeff-phillips-18) [reply frontend/src/pages/documentation/Contribute.js]: the space will be removed when it's at the end of a line.

Comment (jozzi) [reply frontend/src/pages/documentation/Contribute.js]: oh, I wrote "& nbsp;" but it was replaced with space :D

Comment (jozzi) [reply frontend/src/pages/documentation/Contribute.js]: "&nbsp"

Comment (jeff-phillips-18) [reply frontend/src/pages/documentation/Contribute.js]: I've used this method (`{' '}`) throughout so I'd say keep it consistent.

**Pull request merged**